DrinkManagerConsole/drinks_source.json

```diff
@@ -1,2314 +0,0 @@
-{
```

Comment:
Why did you delete this file?

Reply:
I cut-pasted it to new directory, which was specified in Wojtek's PR when he was working on database. It is not necessary in DrinkManagerConsole, so I decided not to leave it there, but as I mentioned above - in my comment about my PR - I can add it if you would prefer it that way.

Reply:
No, you dont't have to add it. I was just curious why this was deleted. :)